docs/specs/Bet/06_Messages.md

```diff
@@ -7,17 +7,17 @@ handler endpoints is as follows
 // Msg defines the Msg service.
 service Msg {
   // PlaceBet defines a method to place a bet with the given data
-  rpc PlaceBet(MsgPlaceBet) returns (MsgPlaceBetResponse);
+  rpc Place(MsgPlace) returns (MsgPlaceResponse);
```

Comment:
rpc Wager is better name which can be used for placing a bet. it is single word and best aligns with the API functionality. Also it is suggested by chatGPT :)